recipes-support/onboard/onboard_%.bbappend

```diff
@@ -0,0 +1,41 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+
+SRC_URI += " \
+	file://0001-add-xfce-to-autostart-onlyshowin.patch \
+	file://0002-onboard-onhover-seg-fault-fix.patch \
+	file://01-gnome-accessibility \
+	file://NI.colors \
+	file://NI.theme \
+	file://onboard-defaults.conf \
+"
+
+CXXFLAGS += "-Werror=declaration-after-statement"
+
+do_install:append () {
+	install -d ${D}${sysconfdir}/dconf/db/local.d
+	install -d ${D}${sysconfdir}/onboard
+	install -d ${D}${datadir}/onboard/themes
+	install -d ${D}${sysconfdir}/xdg/autostart
```

Comment:
I suppose this line isn't required anymore as the folder is already created in the main recipe in meta-oe